errors/apiErrors.ts

```diff
@@ -49,41 +51,39 @@ export function isSpecifiedError(
   );
 }
 
-export function isMissingScopeError(err: Error | AxiosError): boolean {
+export function isMissingScopeError(err: unknown): boolean {
   return isSpecifiedError(err, { statusCode: 403, category: 'MISSING_SCOPES' });
 }
 
-export function isGatingError(err: Error | AxiosError): boolean {
+export function isGatingError(err: unknown): boolean {
   return isSpecifiedError(err, { statusCode: 403, category: 'GATED' });
 }
 
-export function isTimeoutError(err: Error | AxiosError): boolean {
+export function isTimeoutError(err: unknown): boolean {
   return isSpecifiedError(err, { code: 'ETIMEDOUT' });
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function isApiUploadValidationError(err: AxiosError<any>): boolean {
+export function isApiUploadValidationError(err: unknown): boolean {
   return (
-    err.isAxiosError &&
+    isAxiosError(err) &&
     (err.status === 400 || err.response?.status === 400) &&
     !!err.response &&
     !!(err.response?.data?.message || !!err.response?.data?.errors)
   );
 }
 
 export function isSpecifiedHubSpotAuthError(
-  err: GenericError,
+  err: unknown,
   { status, category, subCategory }: Partial<HubSpotAuthError>
-): boolean {
+): err is HubSpotAuthError {
```

Comment:
Wow I did not know you could do this. Huge.

Reply:
Yeah, they are called [type predicates](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) and they are very powerful!